WiFi101WebThingAdapter.h

```diff
@@ -308,30 +308,8 @@ class WebThingAdapter {
             handleError();
           }
           return;
-        } else if (uri == deviceBase + "/properties") {
-          if (method == HTTP_GET || method == HTTP_OPTIONS) {
```

Comment:
Now that we have the root level form, we should keep this.

Reply:
I removed the Wifi101 as a whole. It's a very uncommon board. I don't have it, so I can't even make sure that it actually works. I would like to reduce it to the ESP32, since it is the most used board in the maker scene. And an Arduino library targets mostly this audience. I am able to test the EthernetAdapter though. Not sure, if we should provide an implementation for it and test for it. It just introduces complexity and I am not sure, if it really is worth it.

Reply:
I agree, searching online it seems that WiFi101 is an outdated board, and now it even [officially retired](https://store.arduino.cc/arduino-wifi-101-shield) from the store. On the other hand, I don't want to drop completely support for Arduino-based boards. Newer boards have WiFi built-in and they have their own [WiFi HAL](https://www.arduino.cc/en/Reference/WiFiNINA). We might need to have a look into it in the future.

Reply:
About Ethernet, again it seems pretty old although it seems to be still available in the market. If you can test it I'd put some effort into keeping it. Just put the Ethernet support on a lesser priority level.